src/Interfaces/Options/BackgroundMask/IBackgroundMask.ts

```diff
@@ -0,0 +1,6 @@
+import {IOptionLoader} from "../IOptionLoader";
+
+export interface IBackgroundMask extends IOptionLoader<IBackgroundMask> {
+    cover?: string;
+    enable: boolean;
+}
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [file should end with a newline](https://app.codacy.com/manual/ar3s/tsparticles/pullRequest?prid=5164124)